src/builder/components/question/editable-title/survey-question-editable-title.component.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Input } from 'antd';
+import React, { Component, RefObject } from 'react';
+
+export interface EditableTitleProps {
+    onTitleUpdated: (updatedTitle: string) => void;
+    content: string;
+    heading: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'h7';
```

Comment:
Super small thing, but the html spec only includes up to `h6` 😛  https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement

Reply:
No kidding? My life is a lie. I've sworn by `h7` for the better half of a decade now, lol.